ocamlfind ocamlc                                                             \
           -I +camlp4 dynlink.cma camlp4lib.cma                              \
           unix.cma                                                          \
          `ocamlfind query -r -predicates byte gettext.extract -i-format` \
          `ocamlfind query -r -predicates byte gettext.extract -a-format` \
          `ocamlfind query -r -predicates byte gettext.extract -o-format` \
           Camlp4Bin.cmo                                                     \
          -o ocaml-xgettext
File "/usr/lib64/ocaml/unix.cma(Unix)", line 1:
Warning 31: files /usr/lib64/ocaml/unix.cma(Unix) and /usr/lib64/ocaml/unix.cma(Unix) both define a module named Unix
File "/usr/lib64/ocaml/unix.cma(UnixLabels)", line 1:
Warning 31: files /usr/lib64/ocaml/unix.cma(UnixLabels) and /usr/lib64/ocaml/unix.cma(UnixLabels) both define a module named UnixLabels
File "_none_", line 1:
Error: Some fatal warnings were triggered (2 occurrences)
Makefile:65: recipe for target 'ocaml-xgettext' failed
make[1]: *** [ocaml-xgettext] Error 2
make[1]: Leaving directory '/home/abuild/rpmbuild/BUILD/ocaml-gettext-0.3.5/ocaml-gettext'

---
 ocaml-gettext/Makefile |    1 -
 1 file changed, 1 deletion(-)

--- a/ocaml-gettext/Makefile
+++ b/ocaml-gettext/Makefile
@@ -64,7 +64,6 @@ uninstall: ocaml-xgettext-uninstall
 ocaml-xgettext: $(BUILDBIN)
 	$(OCAMLC)                                                            \
            -I +camlp4 dynlink.cma camlp4lib.cma                              \
-           unix.cma                                                          \
           `$(OCAMLFIND) query -r -predicates byte gettext.extract -i-format` \
           `$(OCAMLFIND) query -r -predicates byte gettext.extract -a-format` \
           `$(OCAMLFIND) query -r -predicates byte gettext.extract -o-format` \
